operators/advanced_network/advanced_network/types.h

```diff
@@ -391,6 +391,8 @@ struct NetworkConfig {
   std::unordered_map<std::string, MemoryRegionConfig> mrs_;
   std::vector<InterfaceConfig> ifs_;
   uint16_t debug_;
+  uint32_t tx_meta_buffers_;
+  uint32_t rx_meta_buffers_;
```

Comment:
Document these (default, reason to update them)

Reply:
This is a generic interface, but I only see it used in DPDK? move it under DPDK specific ones, or ensure it's used in the other backends?

Reply:
don't see docs despite this being resolved?

Reply:
Oops, I thought you meant document in the code which is what that comment was. I'll add it to the docs.